cdc/scheduler/agent.go

```diff
@@ -456,3 +478,23 @@ func (a *BaseAgent) currentOwner() model.CaptureID {
 
 	return a.ownerInfo.OwnerCaptureID
 }
+
+func (a *BaseAgent) resetEpoch() {
+	a.epochMu.Lock()
+	defer a.epochMu.Unlock()
+
+	a.epoch = uuid.New().String()
```

Comment:
We only need the unique constraint of this epoch, and don't need the serialization guarantee. Should we add comment about it.

Reply:
Sure.